### PR TITLE
added FAQ plugin v1

### PR DESCRIPTION
**Pull request by dchicasduena**
Created some commands to answer some questions we always get asked. More could be added for different questions. ( ͡° ͜ʖ ͡°) lmk if I should change anything

### CODE REVIEW

File: plugins/FAQ/__init__.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+            inline=False)
+        embed_science.add_field(
+            name=f'**Winter Semester**',
+            value=f'> Mathematics 1000 or 1001\n> Computer Science 1002\n> Computer Science 1003\n> CRW Course\n> science elective\n',
+            inline=False)
+        embed_science.add_field(
+            name=f'What science electives should I take?',
+            value=f'_The science electives should be in the same discipline (Econ 1010 and Econ 1020). A list of science electives can be found [here](https://www.mun.ca/undergrad/first-year-information/courses-available-in-first-year/)_', 
+            inline=False)
+        embed_science.set_footer(text="Students who have not completed Computer Science 1003 in their first year will not be able to register for Computer Science 2001/2/3 in the fall of their second year.")
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:31]: break into function that returns embed

File: plugins/FAQ/__init__.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+            inline=False)
+        embed_arts.add_field(
+            name=f'**Winter Semester**',
+            value=f'> CRW course\n> Mathematics 1000 or 1001\n> Computer Science 1002\n> Computer Science 1003\n> LS course\n',
+            inline=False)
+        embed_arts.add_field(
+            name=f'What is a Breathe of Knowledge and Language Study elective?',
+            value=f'_"More information for about [Breathe of Knowledge](https://www.mun.ca/regoff/calendar/sectionNo=ARTS-0109#ARTS-8192) and [Language Study](https://www.mun.ca/regoff/calendar/sectionNo=ARTS-0109#ARTS-8196)_', 
+            inline=False)
+        embed_arts.set_footer(text="Students who have not completed Computer Science 1003 in their first year will not be able to register for Computer Science 2001/2/3 in the fall of their second year.")
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:49]: break into function that returns embed

File: plugins/FAQ/__init__.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        embed_science.add_field(
+            name=f'What science electives should I take?',
+            value=f'_The science electives should be in the same discipline (Econ 1010 and Econ 1020). A list of science electives can be found [here](https://www.mun.ca/undergrad/first-year-information/courses-available-in-first-year/)_', 
+            inline=False)
+        embed_science.set_footer(text="Students who have not completed Computer Science 1003 in their first year will not be able to register for Computer Science 2001/2/3 in the fall of their second year.")
+        
+        embed_arts=discord.Embed(
+            title="B.A Sample First Year", url="https://www.mun.ca/undergrad/first-year-information/sample-first-year---st-johns-campus/science/computer-science/", 
+            description="Students pursuing a bachelor of arts with a major in computer science will normally take the following courses in their first year::", 
+            color=0x9898FB)
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:36]: make color a constant defined above

File: plugins/FAQ/__init__.py [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+            name=f'What is a Breathe of Knowledge and Language Study elective?',
+            value=f'_"More information for about [Breathe of Knowledge](https://www.mun.ca/regoff/calendar/sectionNo=ARTS-0109#ARTS-8192) and [Language Study](https://www.mun.ca/regoff/calendar/sectionNo=ARTS-0109#ARTS-8196)_', 
+            inline=False)
+        embed_arts.set_footer(text="Students who have not completed Computer Science 1003 in their first year will not be able to register for Computer Science 2001/2/3 in the fall of their second year.")
+        
+        if degree.upper() in ("BSC", "B.SC", "SCIENCES", "SCIENCE"):
+            await ctx.send(embed=embed_science)
+            
+        elif degree.upper() in ("BA", "B.A", "ART", "ARTS"):
+            await ctx.send(embed=embed_arts)
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:55]: make both of these tuple arrays defined as constants above

File: plugins/FAQ/__init__.py [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+            value=f'In February of each year an online application form will be available on the [Admissions](https://www.mun.ca/computerscience/ugrad/UGProgram/admissions.php) page. This form must be submitted prior to June 1.', 
+            inline=False)
+        embed_admission.add_field(
+            name=f'**What is the minimum required average for acceptance?**',
+            value=f'Students who fulfill the eligibility requirements compete for a limited number of available spaces. Selection is based on academic performance, normally cumulative average and performance in recent courses.  For 2022 applications, students must also have a mean grade of at least 65% in Computer Science 1001 and 1002.', 
+            inline=False)
+      
+        await ctx.send(embed=embed_admission)
+        
+        #( ͡° ͜ʖ ͡°)
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:84]: <3

File: plugins/FAQ/__init__.py [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+            
+        elif degree.upper() in ("BA", "B.A", "ART", "ARTS"):
+            await ctx.send(embed=embed_arts)
+            
+        else:
+            # No type of degree was specified
+            await ctx.send(f"Specify the type of degree (Valid arguments: b.sc, b.a, science, arts)\n")
+
+    @commands.command()
+    async def admission(self, ctx: commands.Context, degree: str = "blank"):
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:62]:
```suggestion
    async def admission(self, ctx: commands.Context):
```
unused field?

**Review by jackharrhy — CHANGES_REQUESTED** [review]

**Comment by dchicasduena** [comment]
newest commit has the revised changes, lmk if I should change anything else

File: plugins/FAQ/__init__.py [hunk 7]
```diff
@@ -48,24 +59,36 @@ async def sample(self, ctx: commands.Context, degree: str = "blank"):
...
-        elif degree.upper() in ("BA", "B.A", "ART", "ARTS"):
-            await ctx.send(embed=embed_arts)
+        elif degree.upper() in ARTS:
+            embed_sample = await self.create_embed_arts()
+            await ctx.send(embed=embed_sample)
             
         else:
             # No type of degree was specified
-            await ctx.send(f"Specify the type of degree (Valid arguments: b.sc, b.a, science, arts)\n")
+            await ctx.send(f"Specify the type of degree (Valid arguments: bsc, b.sc, science, ba, b.a, arts)\n")
```
Comment (jackharrhy) [inline plugins/FAQ/__init__.py:82]: use above constants to generate the valid `(Valid arguments: ... ` string

**Review by dchicasduena — COMMENTED** [review]
:)

**Review by jackharrhy — CHANGES_REQUESTED** [review]
almost gucci, one last thing

**Review by jackharrhy — APPROVED** [review]
not tested, reviewed, great plugin!

**Pull request merged**